src/Bundle/SyliusResourceBundle.php

```diff
@@ -54,6 +55,7 @@ public function build(ContainerBuilder $container): void
         $container->addCompilerPass(new FallbackToKernelDefaultLocalePass());
         $container->addCompilerPass(new DoctrineContainerRepositoryFactoryPass());
         $container->addCompilerPass(new DoctrineTargetEntitiesResolverPass(new TargetEntitiesResolver()), PassConfig::TYPE_BEFORE_OPTIMIZATION, 1);
+        $container->addCompilerPass(new MutatorPass());
```

Comment:
`Mutator` is not specific enough name when used outside of `Sylius\Resource\Metadata` dir.

For example,
```suggestion
        $container->addCompilerPass(new MetadataMutatorPass());
```

Reply:
fixed